roles/upgrade/post-upgrade/tasks/main.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: wait for cilium
+  when:
+    - needs_cordoning|default(false)
+    - kube_network_plugin == 'cilium'
+  command: >
+    {{ bin_dir }}/kubectl --kubeconfig /etc/kubernetes/admin.conf
```

Comment:
Could you replace `/etc/kubernetes/admin.conf` with `{{ kube_config_dir }}/admin.conf` ?

Reply:
Yes, sure